README.md

```diff
@@ -50,6 +50,15 @@ FMOD Studio comes with an Examples project. Open it and build the project. Then
 You can also configure FMOD Studio to build to a folder you specify.
 Run examples with `cargo run --example`. See the source code of the examples for more details.
 
+## Live Update
+
+To enable live update, you need to enable the `live-update` feature. While you can do so in Cargo.toml, I recommend
+to explicitly enable it with the `--features` flag. This way, you won't accidentally include it in your release builds.
+
+```sh
+cargo run --example minimal --features live-update
+```
+
```

Comment:
Maybe put a link in here to https://www.fmod.com/docs/2.01/studio/editing-during-live-update.html for people who don't know about live update.

Reply:
Done ✅